whittle/search/multi_objective.py

```diff
@@ -5,10 +5,15 @@
 
 
 def get_pareto_optimal(costs: np.ndarray) -> npt.NDArray[np.bool_]:
-    """Find the pareto-optimal point.
+    """
+    Find the pareto-optimal point.
+
+    Args:
+        costs: (n_points, m_cost_values) array
```

Comment:
It would be better to be more descriptive here, what do costs indicate? should they be increasing or decreasing?

Reply:
cost is an indication of the validation loss if I am not mistaken so they should be decreasing right?